source/Calamari.Common/CalamariFlavourProgram.cs

```diff
@@ -123,6 +123,7 @@ protected virtual void ConfigureContainer(ContainerBuilder builder, CommonOption
 
             builder.RegisterAssemblyTypes(assemblies)
                 .AssignableTo<IScriptWrapper>()
+                .Where(wrapper => wrapper.IsPublic)
```

Comment:
Instead of modifying this for all the tests let's enable/disable the wrapper on the test itself, see comment below